projects/addon-editor/components/editor-socket/editor-socket.component.ts

```diff
@@ -23,20 +25,24 @@ import {TUI_SANITIZER} from '@taiga-ui/core';
     host: {
         class: 'tui-editor-socket',
     },
+    providers: [TuiDestroyService],
 })
 export class TuiEditorSocketComponent {
     @Input()
+    set editor(editor: AbstractTuiControl<string>) {
```

Comment:
This is wrong, socket is for displaying content that was created by editor, there might be no editor at all when it's used.